### PR TITLE
[Chore] Update for api changes 

### PR DESCRIPTION
**Pull request by KaterinaWire**
## What's new in this PR?

Update for API changes in Request strategy.

Needs releases with:

- [  wireapp/wire-ios-request-strategy/pull/302 ]

### CODE REVIEW

**Comment by typfel** [comment]
So the `Sources/Notifications/Push Notifications/` files wasn't move to request strategy but are actually duplicate at the moment?

**Comment by KaterinaWire** [comment]
> So the `Sources/Notifications/Push Notifications/` files wasn't moved to request strategy but are actually duplicate at the moment?

No, they are not duplicated. Files in the `Sources/Notifications/Push Notifications/` are related to calling.

File: Tests/Source/SessionManager/SessionManagerTests.swift [hunk 1]
```diff
@@ -1083,70 +1083,68 @@ final class SessionManagerTests_MultiUserSession: IntegrationTest {
...
+    //            userSession2 = self.sessionManager!.backgroundUserSessions[account.userIdentifier]
+    //        })
+    //
+    //        XCTAssertTrue(self.waitForCustomExpectations(withTimeout: 0.5))
+    //        XCTAssertNotNil(userSession1)
+    //        XCTAssertNotNil(userSession2)
+    //        XCTAssertEqual(userSession1, userSession2)
+    //        // CLEANUP
+    //        self.sessionManager!.tearDownAllBackgroundSessions()
+    //    }
```
Comment (David-Henner) [inline Tests/Source/SessionManager/SessionManagerTests.swift:1147]: I'd add a TODO to not forget to uncomment this code at some point

**Review by David-Henner — APPROVED** [review]

File: Tests/Source/SessionManager/SessionManagerTests.swift (hunk 1 above)
Comment (KaterinaWire) [reply Tests/Source/SessionManager/SessionManagerTests.swift:1148]: Done

**Pull request merged**